### DISCUSSION TITLE
Come si aiuta?

### DISCUSSION THREAD

**Discussion by EldritchTranslator** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#69] [Domande & Rispsote]
Vorrei aiutare a portare avanti il progetto, possibilmente anche con gli aggiornamenti all'ultima UP inglese, dato che sono una appassionata di VtM e traduttrice. Come ci si organizza? Vengono assegnate delle parti mancanti, e poi si fa un PR? 
1. Come si individuano le linee non tradotte, c'è una lista completa da qualche parte? 
2. Qual è la regola per revisionare traduzioni già presenti ma datate o possibilmente incorrette? 
3. Soprattutto, c'è uno script completo in inglese per controllare le traduzioni? 
4. Ci sono altre linee guida?

**Comment by tesivo** [comment]
Benvenuta, è da un po' che non sto trovando il tempo per portare avanti il progetto. Proverò a dirti, a grandi linee, a che punto siamo.
- la traduzione è completa nel senso che non si ha un'idea precisa di ciò che manca.
- la traduzione va aggiornata all'ultima versione dell'unofficial patch disponibile.
- l'ideale sarebbe di offrire alla traduzione un installer che ne agevoli l'installazione.

Quindi rispondendo alle domande:
1- Giocando, tramite le segnalazioni e confrontando con la base inglese e la traduzione francese.
2- Purtroppo non c'è alcuna regola a riguardo, diciamo che siamo tutti d'accordo di voler tutto subito e perfetto, ma credo sinceramente che possa essere di maggiore interesse l'offrire prima una traduzione completa e solo in un secondo momento precisa.
3- C'è il progetto originale inglese: https://github.com/miracleflame/VTMB-Unofficial-Patch-by-Wesp e la traduzione francese https://github.com/FoolEcho/vtmb-up
4- Qualcosa sì, ma andrebbero riordinate e riscritte per bene, erano frutto di più analisi

**Comment by NarakuITA** [comment]
Benvenuta! Effettivamente c'è bisogno di tutto l'aiuto!
